internal/test/dac/input.cue

```diff
@@ -15,87 +15,93 @@ package test
 
```

Comment:
FYI most relevant file to look at, as it shows the library change from DaC user PoV.